### DISCUSSION TITLE
o.T. (carré noir): Exercise 1

### DISCUSSION THREAD

**Discussion by sharlagelfand** [posit-conf-2023/creative-coding#4] [Show and tell]
1.  Open file: `exercises/03-ot/exercise.Rmd`
2.  On your graphing paper, select from letters in `exercises/03-ot/letters/` and draw out the letter.
3.  Label all the vertices of the shape with the coordinates.
4.  Create a `tribble()` with the coordinates of the letter you selected.
5.  Create a new function, `create_letter()`, that takes `x0` and `y0` as arguments and moves the shape according to those arguments.
6.  Plot your shape at the `x0` and `y0` of your choosing.
7.  Share your output on this Github discussion board.
8.  Remember to save your file when you are done with the exercise.

**Comment by bshk93** [comment]
![ex1-shape](https://github.com/posit-conf-2023/creative-coding/assets/65432465/57c126b1-89fe-4798-81f0-b7f24223a34c)

**Comment by roBertus31** [comment]
![ex1-shape](https://github.com/posit-conf-2023/creative-coding/assets/25853405/411c09c2-91ce-44d0-9afc-98cf3d046872)

**Comment by thomaspara** [comment]
![M](https://github.com/posit-conf-2023/creative-coding/assets/10082327/a71b71f8-eea0-4413-b135-f81ed032f4e6)

**Reply by thomaspara** [reply]
i used 
`
create_letter  <- function(x0 = 1, y0 =4) { letter_tribble %>% mutate(x = x+x0, y = y+y0) }
`

**Comment by Meghansaha** [comment]
![image](https://github.com/posit-conf-2023/creative-coding/assets/67963743/28b16e5b-051e-4141-aff6-da89fb9af809)

**Reply by Meghansaha** [reply]
![image](https://github.com/posit-conf-2023/creative-coding/assets/67963743/5ca33296-f7b1-406d-a7a2-250302454e6e)
 "shifted"

**Comment by Kim-s-h** [comment]
![ex2_1](https://github.com/posit-conf-2023/creative-coding/assets/22897443/5a41b9d1-5419-4e5a-87ef-6723e50dc884)

**Comment by maxrogow** [comment]
![ex1-M-shape](https://github.com/posit-conf-2023/creative-coding/assets/145362959/82f88222-7dd3-4548-b5c0-0aebd7e6e3f1)

**Comment by shaunmwilkinson** [comment]
![image](https://github.com/posit-conf-2023/creative-coding/assets/145362966/871e6eb2-32a4-4b75-895a-cd23014bd9ee)

**Comment by dougtommet** [comment]
![ex1-shape](https://github.com/posit-conf-2023/creative-coding/assets/7944225/d214b8b3-b20f-4b89-8848-796c27aef31c)

**Comment by natalieoshea** [comment]
![image](https://github.com/posit-conf-2023/creative-coding/assets/10425075/f075881e-140d-4c83-9b14-6eb7788fbbd0)

**Comment by roBertus31** [comment]
![ex2-shape](https://github.com/posit-conf-2023/creative-coding/assets/25853405/360b8cbd-ce8e-4e50-841c-a39a6bc04f3f)

**Comment by dougtommet** [comment]
![ex2-shape](https://github.com/posit-conf-2023/creative-coding/assets/7944225/c69a226e-e84e-4304-9445-336a4d21e23f)
